includes/admin/tools/export/class-batch-export-donors.php

```diff
@@ -132,6 +132,33 @@ public function set_properties( $request ) {
 		$this->price_id = ! empty( $request['give_price_option'] ) && 'all' !== $request['give_price_option'] ? absint( $request['give_price_option'] ) : null;
 	}
 
+	/**
+	 * This function is used to define default columns for export.
+	 *
+	 * @since 2.2.6
+	 *
+	 * @return array
+	 */
+	public function get_default_columns() {
```

Comment:
@mehul0810 I can see that you are using this function in HTML view of the export setting page to get donor CSV columns list. I will suggest you create this as an independent function to use it anywhere and add a note in function description that this is only for internal logic and use filter inserted.

Reply:
@ravinderk I've implemented the change as requested.